servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -280,6 +285,9 @@ public void start(boolean isLeader) throws IOException {
       mTTLCheckerService = getExecutorService().submit(
           new HeartbeatThread(HeartbeatContext.MASTER_TTL_CHECK, new MasterInodeTTLCheckExecutor(),
               MasterContext.getConf().getInt(Constants.MASTER_TTLCHECKER_INTERVAL_MS)));
+      mLostFilesDetectionService = getExecutorService().submit(new HeartbeatThread(
+          HeartbeatContext.MASTER_LOST_FILES_DETECTION, new LostFilesDetectionHeartbeatExecutor(),
+          MasterContext.getConf().getInt(Constants.MASTER_HEARTBEAT_INTERVAL_MS)));
```

Comment:
It seems we don't add its own parameter but reuse master heartbeat one except the TTL check. For example, the `LostWorkerDetectionHeartbeatExecutor` in `BlockMaster` uses `MASTER_HEARTBEAT_INTERVAL_MS`, too